src/styles/index.scss

```diff
@@ -2,2 +1,122 @@
+.page {
   margin: 0;
+  padding: 0;
+  font-family: Roboto, sans-serif;
+  color: #060b35;
+
+  scroll-behavior: smooth;
+}
+
+.product-card {
+  display: flex;
+  box-sizing: border-box;
+  width: 200px;
+  background-color: #fff;
+  border: 1px solid #f3f3f3;
+  border-radius: 5px;
+  justify-content: center;
+  flex-direction: column;
+  padding: 32px 16px 16px;
+}
+
+.product-card__image {
+  width: 160px;
+  height: 134px;
+  margin-bottom: 40px;
+  align-self: center;
+  margin-inline: 3px;
+}
+
+.product-card__title {
+  text-align: left;
+  margin-bottom: 4px;
+  font-size: 12px;
+  font-weight: 500;
+  line-height: 18px;
+}
+
+.product-card__code {
+  text-align: left;
+  font-size: 10px;
+  color: #616070;
+
+  font-weight: 400;
+  line-height: 14px;
+  margin-bottom: 16px;
+}
+
+.stars {
+  display: flex;
+  justify-content: space-between;
+  margin-bottom: 24px;
+}
+
+.stars-4 {
+  display: flex;
+  justify-content: space-between;
+  width: 96px;
+}
+
+.stars-4__star {
+  height: 16px;
+  width: 16px;
+  justify-content: center;
+  align-items: center;
+  background-image: url(../images/star.svg);
+  background-repeat: no-repeat;
+  background-position: center;
+  border-radius: 0.5px;
+}
+
+.stars-4__star:nth-child(-n + 4) {
+  background-image: url(../images/star-active.svg);
+}
```

Comment:
The use of `:nth-child(-n + 4)` is a clever way to apply styles to the first four elements. However, ensure this is flexible enough to accommodate different numbers of active stars if the number of stars needs to be dynamic.